<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtAddToolGUI</class>
 <widget class="QWidget" name="QtAddToolGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>657</width>
    <height>512</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>657</width>
    <height>512</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>657</width>
    <height>512</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>QtAddToolGUI</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QTabWidget" name="tabWidget_Tools">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab_basicT">
        <attribute name="title">
         <string>Basic tools</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="PButtn_outline">
             <property name="text">
              <string>Outline</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="PButtn_color">
             <property name="text">
              <string>Color</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="PButton_edgePix">
             <property name="text">
              <string>Edge Pixel</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <widget class="Line" name="line">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="PButton_positAdjust">
           <property name="text">
            <string>Position Adjustmen</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_advansT1">
        <attribute name="title">
         <string>Advansed Tools 1</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>112</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="PButton_width">
           <property name="text">
            <string>Width</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="PButton_diametr">
           <property name="text">
            <string>Diametr</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="PButton_edge">
           <property name="text">
            <string>Edge</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="PButton_pitch">
           <property name="text">
            <string>Pitch</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_4">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>111</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="Line" name="line_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="PButton_highSpeedPosit">
           <property name="text">
            <string>High-Speed position Adjustment</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_advansT2">
        <attribute name="title">
         <string>Advansed tools 2</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QPushButton" name="PButton_OCR">
           <property name="text">
            <string>OCR</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_5">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>357</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QStackedWidget" name="stackedWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="pag_BasT_Outlin">
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>280</y>
           <width>261</width>
           <height>71</height>
          </rect>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Outline calculates the maching % of a target by&lt;/p&gt;&lt;p&gt; comparing the curent outline to an outline stoed &lt;/p&gt;&lt;p&gt;as a master reference. &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="pag_BasT_Color">
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>280</y>
           <width>261</width>
           <height>71</height>
          </rect>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Color calculates the maching % of a target by&lt;/p&gt;&lt;p&gt;comparing the number of pixels in current window&lt;/p&gt;&lt;p&gt;to the same window on the master image.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="pag_BasT_Edge">
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>300</y>
           <width>261</width>
           <height>71</height>
          </rect>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Judges difference in no. of pixels in an edge &lt;/p&gt;&lt;p&gt;based on no. of pixels in the edge registered&lt;/p&gt;&lt;p&gt;in the master image. &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="pag_BasT_PositAdj">
        <widget class="QLabel" name="label_5">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>270</y>
           <width>261</width>
           <height>51</height>
          </rect>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Position Adjustment allows tools to follow a key &lt;/p&gt;&lt;p&gt;feature (pattem) on the target.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="pag_AdvT1_Width">
        <widget class="QLabel" name="label_6">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>340</y>
           <width>261</width>
           <height>51</height>
          </rect>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Judges difference in length of target compared &lt;/p&gt;&lt;p&gt;to width in master image.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="pag_AdvT1_Diametr">
        <widget class="QLabel" name="label_7">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>350</y>
           <width>261</width>
           <height>51</height>
          </rect>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Judges difference in diametr on target compared &lt;/p&gt;&lt;p&gt;to diametr in master image.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="pag_Adv1_Edge">
        <widget class="QLabel" name="label_8">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>320</y>
           <width>261</width>
           <height>51</height>
          </rect>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Counts number of edges present on target.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="pag_Adv1_Pitch">
        <widget class="QLabel" name="label_9">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>300</y>
           <width>271</width>
           <height>91</height>
          </rect>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Judges difference in pitch width and number &lt;/p&gt;&lt;p&gt;of pins on target compared to value of all &lt;/p&gt;&lt;p&gt;pitch widths in the master image.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="page_Adv1_HighSpeed">
        <widget class="QLabel" name="label_10">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>100</y>
           <width>261</width>
           <height>51</height>
          </rect>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Adjusts for displacement in 1 axis &lt;/p&gt;&lt;p&gt;direction at high speeds.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_11">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>350</y>
           <width>261</width>
           <height>51</height>
          </rect>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Adjusts for displacement in X and Y &lt;/p&gt;&lt;p&gt;direction at high speeds.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="pag_Adv2_OCR">
        <widget class="QLabel" name="label_12">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>310</y>
           <width>261</width>
           <height>51</height>
          </rect>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Judge the target based on the characters/date&lt;/p&gt;&lt;p&gt;in the master image.&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_OK">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Canel">
       <property name="text">
        <string>Canel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
